src/libraries/System.Console/src/System/Console.cs

```diff
@@ -379,16 +379,23 @@ public static bool CursorVisible
 
         public static int CursorLeft
         {
-            get { return ConsolePal.CursorLeft; }
+            get { return ConsolePal.GetCursorPosition().Left; }
             set { SetCursorPosition(value, CursorTop); }
         }
 
         public static int CursorTop
         {
-            get { return ConsolePal.CursorTop; }
+            get { return ConsolePal.GetCursorPosition().Top; }
             set { SetCursorPosition(CursorLeft, value); }
         }
 
+        /// <summary>Gets the position of the cursor.</summary>
+        /// <returns>The column and row position of the cursor.</returns>
```

Comment:
Should it doc whether it's zero based? (And note that it's from left and top, not sure whether intellisense shows the return tuple param names conveniently)

Reply:
https://github.com/dotnet/runtime/pull/37609